<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab5_team12_KA64</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>891</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_00"/>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="lineEdit_24"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_33"/>
     </item>
     <item row="3" column="6">
      <widget class="QLineEdit" name="lineEdit_36"/>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="lineEdit_25"/>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="lineEdit_16"/>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="lineEdit_15"/>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="lineEdit_35"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="lineEdit_34"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_20"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_31"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_23"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_21"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_22"/>
     </item>
     <item row="2" column="6">
      <widget class="QLineEdit" name="lineEdit_26"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_32"/>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="lineEdit_05"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_30"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_02"/>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="lineEdit_04"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_01"/>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="lineEdit_06"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_03"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>571</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="line_input">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Input file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="select_input">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Величина допуску:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="CalculateButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Порахувати</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GrapfButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Вивести графіки</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Результати обчислень:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>881</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_s1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>для ситуації S3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>для ситуації S2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>для ситуації S4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>10</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>для ситуації S1 </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_s2"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_s3"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_s4"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
